src/main/java/Miku.java

```diff
@@ -0,0 +1,126 @@
+import java.util.Scanner;
+
+public class Miku {
+
+    static Task[] storedList = new Task[100];
```

Comment:
You could consider refactoring the magic number to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)